src/psyclone/nemo.py

```diff
@@ -519,8 +506,9 @@ class NemoImplicitLoop(NemoLoop):
 
     '''
     def __init__(self, ast, parent=None):
+        valid_loop_types = Config.get().api_conf("nemo").get_valid_loop_types()
         Loop.__init__(self, parent=parent,
-                      valid_loop_types=VALID_LOOP_TYPES)
+                      valid_loop_types=valid_loop_types)
```

Comment:
Again, I think we're passing a dict as a list here.

Reply:
Done